bin/content-push

```diff
@@ -14,12 +14,28 @@ def ask_for_name
   name
 end
 
+def disallow_case_change
+  `git status --porcelain --untracked-files=all #{CONTENT_PATHS}`.split("\n").each do |line|
+    if (%r{^R  ([^ ]+) -> ([^ ]+)}.match(line) || %r{^R  ("[^"]+") -> ("[^"]+")}.match(line)) && $1.casecmp($2) == 0
```

Comment:
Are there always the same number of spaces? Safe to use this over whitespace: `\s+`?